code/game/turfs/simulated/walls.dm

```diff
@@ -86,7 +86,8 @@
 
 	if(rotting)
 		. += "<span class='warning'>There is fungus growing on [src].</span>"
-	. += "<span class='notice'>Using a lit welding tool on this item will allow you to slice through it, eventually removing the outer layer.</span>"
+	if(can_dismantle_with_welder)
+		. += "<span class='notice'>Using a lit welding tool on this item will allow you to slice through it, eventually removing the outer layer.</span>"
```

Comment:
We would want reinforced walls to have this message right? Because they have this var set as false for a reason that is not "indestructible wall"

Reply:
ugh, yeah good catch. I will sort that out tonight. dammit I thought this was a simple fix!

Reply:
We all do 😢

Reply:
as it turns out, not a worry.